sipa/utils/__init__.py

```diff
@@ -159,6 +159,25 @@ def meetingcal():
     return next_meetings
 
 
+def subscribeStatusPage(url: str, email: str) -> bool | None:
+    try:
+        response = requests.post(url, timeout=1,
```

Comment:
It might be a good idea to make the timeout configurable as well if you finish the feature